src/main/java/org/spongepowered/mod/service/world/SpongeChunkTicketManager.java

```diff
@@ -178,6 +180,11 @@ public int getMaxNumChunks() {
         }
 
         @Override
+        public World getWorld() {
+            return this.world;
+        }
```

Comment:
I've gone through the process again, updating the submodules to point to my forked code; if you want you can disregard this pull and I can make another; or you can pull this and once the API is accepted, it will work.